RockWeb/Blocks/Finance/TransactionEntry.ascx.cs

```diff
@@ -312,6 +324,55 @@ protected override void OnLoad( EventArgs e )
             pnlDupWarning.Visible = false;
             nbSaveAccount.Visible = false;
 
+            string accountParameterType = string.Empty;
+            using ( var rockContext = new RockContext() )
+            {
+                if ( !string.IsNullOrWhiteSpace( PageParameter( "account" ) ) )
+                {
+                    List<int> accountParameter = new List<int>();
+                    accountParameterType = "invalid";
+                    foreach ( string acctId in PageParameter( "account" ).Split( ',' ) )
+                    {
+                        int id = 0;
```

Comment:
instead of TryParse, you could simply do int accountId = acctId.AsInteger(), or even do a foreach (int accountId in PageParameter("AccountIds").Split(',').AsIntegerList())